src/preferences.html

```diff
@@ -0,0 +1,222 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="Content-Security-Policy" content="default-src 'self'; style-src 'self'; script-src 'unsafe-inline'">
+  <link rel="stylesheet" href="./assets/css/bulma.min.css">
+  <link rel="stylesheet" href="./assets/css/style.css">
+  <title>Preferences</title>
+</head>
+
+<body>
+
+  <header>
+
+      <nav class="navbar" role="navigation" aria-label="main navigation">
+        <div class="navbar-brand">
+          <div class="navbar-item">
+            <img src="./icons/streaker-icon.svg" alt="Streaker Logo">
```

Comment:
Now that the navbar will be dark we'll need a white icon for the header.

I think that we could take a copy of `src/icons/macos/doneTemplate@2x.png` and recolour it to be white. This icon is a good size for a title bar icon and already has retina support.